includes/settings/class-wcpdf-settings-callbacks.php

```diff
@@ -714,6 +717,23 @@ public function validate( $input ) {
 		// Return the array processing any additional functions filtered by this action.
 		return apply_filters( 'wpo_wcpdf_validate_input', $output, $input );
 	}
+
+	/**
+	 * Appends language at the end of the setting provided, in case the setting is translatable
+	 * and it does not have a language set.
+	 *
+	 * @param string $setting Settings field that needs a language.
+	 * @param array  $args Setting arguments.
+	 *
+	 * @return string
+	 */
+	public function append_language( string $setting, array $args ): string {
+		if ( isset( $args['translatable'] ) && isset( $args['lang'] ) && 'default' !== $args['lang'] && true == $args['translatable'] && ! ( substr( $setting, -strlen( "[{$args['lang']}]" ) ) === "[{$args['lang']}]" ) ) {
+			return $setting .= "[{$args['lang']}]";
```

Comment:
@MohamadNateqi Can you be more specific, I am not sure I understand what do you mean.
If you mean that it is not necessary to use append language function on the `$setting_name `variable, this is necessary because after I set the new image as logo, the media_callback function will not automatically assign the language to the settings name, and therefore, the setting will be saved in the database in the wrong way.

Reply:
Sure, sorry.
I meant you can directly return the value, I mean like this:
```php
return "[{$args['lang']}]";
```

Reply:
@MohamadNateqi But then, only the language will be returned. I need to return the setting name with the language appended at the end. I think this needs to stay as is. Am I missing something?

Reply:
Sorry, I missed `$setting` in my previous code. I meant this:
```php
return $setting . "[{$args['lang']}]";
```
But not important, no need to change.